### ISSUE TITLE
false positive for no-lonely-if 

### ISSUE THREAD

**Issue by bertyhell** [eslint/eslint#5665]
![image](https://cloud.githubusercontent.com/assets/1710840/14023860/b7a22bd8-f1e6-11e5-8ace-82fc8c540461.png)

```javascript
if (angular.isUndefined(attrs.model)) {
  utilitiesService.safeApply(function() {
    scope.item = scope.$parent.item;
    scope.pane = {data: scope.$parent.item};
  }, scope);
} else {
  scope.$watch('model', function(newval) {
    if (newval) {
      utilitiesService.safeApply(function() {
        scope.item = newval;
        scope.pane = {data: newval};
      }, scope);
    }
  });
}
```

This is not a lonely if, but the error still shows up.

eslint version: 2.4.0

**Comment by michaelficarra** [comment]
Cannot reproduce. Please use the issue template next time.

**Comment by eslintbot** [comment]
Hi @bertyhell, thanks for the issue. It looks like there's not enough information for us to know how to help you. If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by alberto** [comment]
I can't reproduce it either, so closing. Please reopen if you can provide more info.
